src/main/java/edu/greenblitz/robotName/subsystems/arm/EndEffector/WristUtils/NeoWrist/NeoWrist.java

```diff
@@ -21,21 +17,23 @@ public class NeoWrist implements IWrist {
 
 
     public NeoWrist() {
-        motor = new GBSparkMax(NeoWristConstants.PID_SLOT, CANSparkMaxLowLevel.MotorType.kBrushless);
+        motor = new GBSparkMax(NeoWristConstants.MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless);
         motor.config(NeoWristConstants.WRIST_CONFIG_OBJECT);
 
-        motor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle).setPositionConversionFactor(WristConstants.ABSOLUTE_POSITION_CONVERSION_FACTOR);
-        motor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle).setVelocityConversionFactor(WristConstants.ABSOLUTE_VELOCITY_CONVERSION_FACTOR);
+        motor.getPIDController().setFeedbackDevice(motor.getEncoder());
 
-        motor.getEncoder().setPositionConversionFactor(RELATIVE_POSITION_CONVERSION_FACTOR);
-        motor.getEncoder().setVelocityConversionFactor(RELATIVE_VELOCITY_CONVERSION_FACTOR);
+        motor.getEncoder().setPositionConversionFactor(CONVERSION_FACTOR);
+        motor.getEncoder().setVelocityConversionFactor(CONVERSION_FACTOR);
 
-        motor.getPIDController().setFeedbackDevice(motor.getAbsoluteEncoder(SparkMaxAbsoluteEncoder.Type.kDutyCycle));
         motor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kForward, true);
         motor.enableSoftLimit(CANSparkMax.SoftLimitDirection.kReverse, true);
```

Comment:
change the 'true' of the is enabled of the limit to be 2 constants, one for the forward and one for the backward